apps/dashboard/src/components/subscribers/locale-select.tsx

```diff
@@ -52,20 +57,22 @@ export function LocaleSelect({
           </div>
         </SelectValue>
       </SelectTrigger>
-      <SelectContent className="p-1">
-        {locales.map((item) => (
-          <SelectItem key={item.langIso} value={item.langIso} className="px-2">
-            <div className="flex w-full items-center gap-2">
-              <div>
-                <CountryFlags name={item.alpha2} className="size-4" />
+      {open && (
+        <SelectContent className="p-1">
```

Comment:
IDK why but the list is not scrollable with the mouse :/ and is a bit lagging on open


https://github.com/user-attachments/assets/bcad787f-26c1-4f0d-bb20-c48f002c0f8d

Reply:
Scrollable will be fixed, was messing around with `modal` and messed it up :P. I'll write a detailed explanation about the pr in a bit.